Essentials/src/com/earth2me/essentials/Settings.java

```diff
@@ -976,6 +978,36 @@ public boolean areDeathMessagesEnabled() {
         return config.getBoolean("death-messages", true);
     }
 
+    private String vanishingItemPolicy;
+
```

Comment:
Might be a bit cleaner with a `KeepInvPolicy` enum rather than just strings?

Reply:
Not sure if there's much point to that, since otherwise it's just trying to parse strings to enums.

Reply:
@pop4959 I'd prefer an enum to magic values being used to determine behaviour, even if those magic values are strings.

Reply:
Nothing really wrong with it, just adds an extra layer of code. Doesn't have any impact on the user, since they're still typing in the same strings.